app/javascript/components/tooltips/Tooltip.tsx

```diff
@@ -262,6 +247,28 @@ export const Tooltip = ({
     modifiers: [{ name: 'offset', options: { offset: [0, 8] } }],
   })
 
+  // Retrieve the HTML contents from the contentEndpoint
+  const { isLoading, isError, data: htmlContent } = useQuery<string>(
+    id,
+    () => {
+      const controller = new AbortController()
+      const signal = controller.signal
+      return Object.assign(
+        // Create a fetch request for the tooltip content, assign the abort controller to the promise
+        // https://react-query.tanstack.com/docs/guides/query-cancellation#using-fetch
+        fetch(contentEndpoint, {
+          method: 'get',
+          signal,
+        }).then((res) => res.text()),
+        { cancel: () => controller.abort() }
+      )
+    },
+    {
+      // Enable the query to fetch only if it isn't in a hidden or error state
+      enabled: !(showState == 'hidden' || showState == 'error'),
```

Comment:
This is something different than: don't fire on mount.

Your implementation sounds like the right approach for the problem, but double confirming that you're not actually looking to make it "not fire on mount" but rather "lazily fire".

Not fire on mount has a dedicated option in react query

Reply:
yes, I'm looking to lazily fire.  I didn't see that option.  Will look again

Reply:
I don't see that api call, or another option.  Do you have a reference I can review?

Reply:
What you're doing right now is lazily fire. This PR is called "shouldn't fire on mount".

Your code does what you want it to do, but the PR is named incorrectly. My comment was trying to figure out if the code was incorrect or the PR was incorrect. The "shouldn't fire on mount" option is called `refetchOnMount`.